src/app/frontend/deploymentdetail/deploymentdetail.html

```diff
@@ -65,5 +65,20 @@
   </kd-content>
 </kd-content-card>
 
+<kd-content-card>
+  <kd-title>[[Horizontal Pod Autoscalers|Title for the 'Horizontal Pod Autoscalers' card on the Replication Controller detail page]]
```

Comment:
Ack, somehow this got changed between me writing the code and this review getting there. Also somehow that change got merged very quickly ;)